bin/apisix

```diff
@@ -44,6 +45,16 @@ else
 end
 -- print("apisix_home: ", apisix_home)
 
+do
+    local ok, json = pcall(require, "cjson")
+    if ok and json then
```

Comment:
It's wrong when install APISIX using PRM package, which will run `bin/apisix` using openresty's luajit.

Reply:
the openresty's luajit can not load the cjson library in openresty too. so the current way is safe.

```lua
$ /usr/local/openresty/luajit/bin/luajit
LuaJIT 2.1.0-beta3 -- Copyright (C) 2005-2017 Mike Pall. http://luajit.org/
JIT: ON SSE2 SSE3 SSE4.1 BMI2 fold cse dce fwd dse narrow loop abc sink fuse
> require("cjson")
stdin:1: module 'cjson' not found:
	no field package.preload['cjson']
	no file './cjson.lua'
	no file '/usr/local/openresty/luajit/share/luajit-2.1.0-beta3/cjson.lua'
	no file '/usr/local/share/lua/5.1/cjson.lua'
	no file '/usr/local/share/lua/5.1/cjson/init.lua'
	no file '/usr/local/openresty/luajit/share/lua/5.1/cjson.lua'
	no file '/usr/local/openresty/luajit/share/lua/5.1/cjson/init.lua'
	no file './cjson.so'
	no file '/usr/local/lib/lua/5.1/cjson.so'
	no file '/usr/local/openresty/luajit/lib/lua/5.1/cjson.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	stdin:1: in main chunk
	[C]: at 0x00405390
```